spec/abci++/abci++_tmint_expected_behavior_002_draft.md

```diff
@@ -145,33 +141,35 @@ Let us now examine the grammar line by line, providing further details.
 >consensus-exec      = (inf)consensus-height
 >```
 
-* A consensus height consists of zero or more rounds before deciding via a call to `FinalizeBlock`.
-  In each round, the sequence of method calls depends on whether the local process is the proposer or not.
+* A consensus height consists of zero or more rounds before deciding via a call to `FinalizeBlock`,
+  followed by a call to `Commit`. If the height has zero rounds, this means the process is
+  replaying an already decided value (catch-up mode). In each round, the sequence of method calls
```

Comment:
Is the sentence starting with "In each round" meant to be distinct from the previous sentence? As in, it is not related to catch-up mode?

Reply:
Correct. I see what you mean. I'll reverse the order of both sentences to reduce ambiguity